src/components/Navbar/Navbar.scss

```diff
@@ -0,0 +1,22 @@
+.navbar {
+  display: flex;
+  justify-content: space-between;
+  box-shadow: 0px 4px 6px rgba(128, 126, 126, 0.1);
```

Comment:
If it's a rudimentary style, then leave it. Else, **ALWAYS** use relative units like rem or vh